activesupport/lib/active_support/core_ext/enumerable.rb

```diff
@@ -60,6 +60,19 @@ def many?
   def exclude?(object)
     !include?(object)
   end
+
+  # Returns a copy of the enumerable without the specified elements.
+  #
+  #   people = ["David", "Rafael", "Aaron", "Todd"]
+  #   people.without "Aaron", "Todd"
+  #     => ["David", "Rafael"]
```

Comment:
No worries! Happy to change this to be more consistent with preferred style in another PR :wink: